### PR TITLE
Use new orchestration status endpoint to display subsystem statuses

### PR DESCRIPTION
**Pull request by rushtong**
Addresses https://broadinstitute.atlassian.net/browse/GAWB-1979

An example of the error case:
error_case

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by davidangb** [comment]
I'd prefer that the UI look at the `systems` part of the API response, and loop over every key found in there, instead of hardcoding the existence of rawls/agora/thurloe/search. That way, the API can add or remove subsystems and the UI will adapt/not break. Plus, it could make the UI code smaller!

File: src/cljs/main/broadfcui/page/status.cljs [hunk 1]
```diff
@@ -3,56 +3,63 @@
...
       [:div {}
-       [StatusLine {:label "Orchestration" :request {:path "/status/ping"}}]
-       [StatusLine {:label "Rawls" :request {:path "/workspaces"}}]
-       [StatusLine {:label "Agora" :request {:path "/methods"}}]
-       [StatusLine {:label "Thurloe" :request {:path "/profile"}}]]])})
+        [StatusLine {:label "Orchestration" :success? orch-ok?    :errors orch-errors}]
+        [StatusLine {:label "Rawls"         :success? rawls-ok?   :errors rawls-errors}]
+        [StatusLine {:label "Agora"         :success? agora-ok?   :errors agora-errors}]
+        [StatusLine {:label "Thurloe"       :success? thurloe-ok? :errors thurloe-errors}]
+        [StatusLine {:label "Search"        :success? search-ok?  :errors search-errors}]]]))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/status.cljs]: For Rawls, Agora, Thurloe, and Search, try something like:

```clojure
(map (fn [system]
        (let [{:keys [messages ok]} (get-in parsed-response [:systems system])]
          [StatusLine {:label (name system) :success? ok :errors messages}]
     (:systems parsed-response))
```

Comment (rushtong) [reply src/cljs/main/broadfcui/page/status.cljs]: Thank you! I was trying to write something like that based on David's feedback.

**Comment by MattPutnam** [comment]
Some indentation issues: vectors (e.g. DOM elements) should have one-space indenting:
```clojure
[:parent {}
 [:child {} ... ]]
```
Other than that this looks great

**Review by davidangb — APPROVED** [review]
👍 lgtm!

**Pull request merged**